sass/layout/_sidebar-left.scss

```
@@ -860,6 +966,18 @@
         height: 18px;
     }
 
+    .SidebarChannelLinkLabel_wrapper {
+        text-overflow: ellipsis;
+        overflow: hidden;
+        display: flex;
+        margin-right: 16px;
+
+        > div {
+            text-overflow: ellipsis;
+            overflow: hidden;
+        }
+    }
+
```

Looks like this has a span as the child element instead of a div on the spinmint, so the ellipsis is not showing in the channel label. Also, the `text-overfow: ellipsis` is not needed on the wrapper element, just the child element.

```suggestion
    .SidebarChannelLinkLabel_wrapper {
        overflow: hidden;
        display: flex;
        margin-right: 16px;

        > span {
            text-overflow: ellipsis;
            overflow: hidden;
        }
    }

```